consensus/src/round_manager.rs

```diff
@@ -1007,22 +1107,73 @@ impl RoundManager {
                         BlockStage::QC_AGGREGATED,
                     );
                 }
-                self.new_qc_aggregated(qc, vote.author()).await
+                QC_AGGREGATED_FROM_VOTES.inc();
+                let result = self.new_qc_aggregated(qc.clone(), vote.author()).await;
+                if self.onchain_config.order_vote_enabled() {
+                    if result.is_ok() {
+                        let _ = self.broadcast_order_vote(vote, qc.clone()).await;
```

Comment:
the error is ignored here

Reply:
I updated this line to have a warn! statement when broadcasting order votes fail. But I didn't have the process_vote_reception_result function to return error when broadcast order vote fails, as order votes are an optimization and the older execution should still succeed even if broadcasting order votes fail.